src/utils/release_activity_updater.rs

```diff
@@ -1,42 +1,50 @@
-
 use crate::db::connect_db;
-use time::{now, Duration};
-use std::collections::BTreeMap;
-use rustc_serialize::json::ToJson;
 use crate::error::Result;
-
+use rustc_serialize::json::ToJson;
+use std::collections::BTreeMap;
+use time::{now, Duration};
 
 pub fn update_release_activity() -> Result<()> {
-
     let conn = connect_db()?;
-    let mut dates = Vec::new();
-    let mut crate_counts = Vec::new();
-    let mut failure_counts = Vec::new();
+    let mut dates = Vec::with_capacity(30);
+    let mut crate_counts = Vec::with_capacity(30);
+    let mut failure_counts = Vec::with_capacity(30);
```

Comment:
Ooh this is nice, I'm surprised clippy was this smart.

Reply:
Well my friend, that wasn't clippy